gtsfm/two_view_estimator.py

```diff
@@ -316,6 +316,11 @@ def aggregate_frontend_metrics(
     """
     num_image_pairs = len(two_view_reports_dict.keys())
 
+    inlier_ratio_gt_model_all_pairs = [report.inlier_ratio_gt_model for report in two_view_reports_dict.values()]
+    inlier_ratio_est_model_all_pairs = [report.inlier_ratio_est_model for report in two_view_reports_dict.values()]
+    num_inliers_gt_model_all_pairs = [report.num_inliers_est_model for report in two_view_reports_dict.values()]
+    num_inliers_est_model_all_pairs = [report.num_inliers_gt_model for report in two_view_reports_dict.values()]
```

Comment:
I think it's better to iterate over two_view_reports_dict once instead of four times. Not a bottleneck, but seems a bit odd.

Reply:
thanks for taking a look @akshay-krishnan.

I agree it's a bit weird, but it was 4 lines instead of 9, so i sort of like it. But I'm good with whatever you prefer here

```python
inlier_ratio_gt_model_all_pairs = []
inlier_ratio_est_model_all_pairs = []
num_inliers_gt_model_all_pairs = []
num_inliers_est_model_all_pairs = []
for report in two_view_reports_dict.values()
    inlier_ratio_gt_model_all_pairs.append(report.inlier_ratio_gt_model)
    inlier_ratio_est_model_all_pairs.append(report.inlier_ratio_est_model)
    num_inliers_gt_model_all_pairs.append(report.num_inliers_est_model)
    num_inliers_est_model_all_pairs.append(report.num_inliers_gt_model)
```
vs.

```python
    inlier_ratio_gt_model_all_pairs = [report.inlier_ratio_gt_model for report in two_view_reports_dict.values()]
    inlier_ratio_est_model_all_pairs = [report.inlier_ratio_est_model for report in two_view_reports_dict.values()]
    num_inliers_gt_model_all_pairs = [report.num_inliers_est_model for report in two_view_reports_dict.values()]
    num_inliers_est_model_all_pairs = [report.num_inliers_gt_model for report in two_view_reports_dict.values()]
```

Reply:
I agree we are getting a lot of metrics now, but I think these metrics are some of the most important for further improvement, so I vote we include them

Reply:
> I agree it's a bit weird, but it was 4 lines instead of 9, so i sort of like it. But I'm good with whatever you prefer here

Isnt it better to have a 4x computational improvement than a 2x code length improvement? In my opinion, it is not  even a comparison we should be making.

Reply:
@akshay-krishnan that's a good point. I updated it, should be cleaner now.

Reply:
turns out the weird phenomenon was due to a simple bug:
```
    num_inliers_gt_model_all_pairs = [report.num_inliers_est_model for report in two_view_reports_dict.values()]
    num_inliers_est_model_all_pairs = [report.num_inliers_gt_model for report in two_view_reports_dict.values()]
```
incorrect assignment